extensions/curator/curator-docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+
+services:
+
+
+  curator:
+    build:
+      context: extensions/curator/
+    environment:
+      ELASTICSEARCH_HOST: "elasticsearch"
```

Comment:
Best practice: avoid quotes in YAML when not necessary, double quotes are especially evil in some circumstances.